frameworks/non-keyed/aurelia2/vite.config.js

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vite';
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
+import aurelia from '@aurelia/vite-plugin';
+import babel from 'vite-plugin-babel';
+
+export default defineConfig({
+  server: {
+    open: !process.env.CI,
+    port: 9000,
+  },
+  base: '/frameworks/non-keyed/aurelia2/dist/',
```

Comment:
I had to add this or when the built app was run, it was trying to reference the bundled assets from the root, instead of using the `aurelia2/dist` directory as the root.